### ISSUE TITLE
gfortran 4.8.5 complains of uninitialised variables

### ISSUE THREAD

**Issue by fcasson** [QuaLiKiz-group/QuaLiKiz#35]
In the current JETTO version

https://git.ccfe.ac.uk/jintrac/jetto-sanco/-/jobs/95278

``
mpif90 -c -O2 -fdefault-real-8 -fdefault-double-8 -fno-align-commons -m64 -mcmodel=medium -fbackslash -Wuninitialized -Wno-maybe-uninitialized -Werror -fopenmp   slatecroutines.f
slatecroutines.f: In function 'i1mach':
slatecroutines.f:1716:0: error: 'equiv.0.imach[_5]' is used uninitialized in this function [-Werror=uninitialized]
       I1MACH=IMACH(I)

 ^
f951: all warnings being treated as errors
``
We can workaround it by removing -Werror, but we'd prefer not to.

**Comment by Karel-van-de-Plassche** [comment]
I think for now it's best to remove -Werror, as I think there are more warnings in the QuaLiKiz compilation right now. After the finalization of the JETTO-QuaLiKiz interface and merge of QuaLiKiz-JETTO version and QuaLiKiz mainline we can revisit the compilation warnings. What do you think @jcitrin?

**Comment by jcitrin** [comment]
Yes agreed. Unfortunately it's difficult to find time at the moment but we need to clean up the code following revisiting the compilation warnings. Good timing could be when the NAG routines are replaced by the open-source version, which is getting closer
